chainer/functions/connection/n_step_lstm.py

```diff
@@ -117,33 +117,92 @@ def _split(inputs, pos):
     return inputs[:pos], inputs[pos:]
 
 
-class NStepLSTM(function.Function):
-
-    def __init__(self, n_layers, states, train=True):
+if cuda.cudnn_enabled and _cudnn_version >= 5000:
+    # Map string names to enums.
+    # Keep enums in the same dict, so that interfaces support both.
+    _rnn_dirs = {
+        'uni': libcudnn.CUDNN_UNIDIRECTIONAL,
+        'bi':  libcudnn.CUDNN_BIDIRECTIONAL,
+        libcudnn.CUDNN_UNIDIRECTIONAL: libcudnn.CUDNN_UNIDIRECTIONAL,
+        libcudnn.CUDNN_BIDIRECTIONAL: libcudnn.CUDNN_BIDIRECTIONAL,
+    }
+
+    _rnn_modes = {
+        'relu': libcudnn.CUDNN_RNN_RELU,
+        'tanh': libcudnn.CUDNN_RNN_TANH,
+        'gru': libcudnn.CUDNN_GRU,
+        'lstm': libcudnn.CUDNN_LSTM,
+        libcudnn.CUDNN_RNN_RELU: libcudnn.CUDNN_RNN_RELU,
+        libcudnn.CUDNN_RNN_TANH: libcudnn.CUDNN_RNN_TANH,
+        libcudnn.CUDNN_GRU: libcudnn.CUDNN_GRU,
+        libcudnn.CUDNN_LSTM: libcudnn.CUDNN_LSTM
+
+    }
+
+    _rnn_params_modes = {
+        # Todo: check this is ok.
+        libcudnn.CUDNN_RNN_RELU: {'n_W': 2, 'n_Wb': 2 * 2, 'n_cell': 1},
+        libcudnn.CUDNN_RNN_TANH: {'n_W': 2, 'n_Wb': 2 * 2, 'n_cell': 1},
+        libcudnn.CUDNN_GRU: {'n_W': 6, 'n_Wb': 2 * 6, 'n_cell': 1},
+        libcudnn.CUDNN_LSTM: {'n_W': 8, 'n_Wb': 2 * 8, 'n_cell': 2},
+        libcudnn.CUDNN_UNIDIRECTIONAL: {'n': 1},
+        libcudnn.CUDNN_BIDIRECTIONAL: {'n': 2}
+    }
+
+
+class NStepRNN(function.Function):
```

Comment:
Is it possible to remove `NStepRNN` from `n_step_lstm.py` and create new file for it, as `NStepGRU` and `NStepBiGRU` also use it?